src/components/d3-component/d3-component.jsx

```diff
@@ -99,23 +89,21 @@ class D3Component extends Component {
       .width(this.props.width)
       .height(this.props.height);
 
-    // @todo: Durran add the original type here.
+    // @todo: Durran: add the original type here.
+    //
+    // @todo: Irina: figure out if we need the promoter, since all the values
+    // are already converted to acceptable JS values. bson_type can be stored in
+    // options as well
     this.state.chart.options({
       fieldName: this.props.fieldName,
-      unique: this.props.type.unique,
+      unique: this.props.type.unique || 0,
       query: this.props.query,
-      promoter: TO_BSON_CONVERSIONS[this.props.type.bsonType] || DEFAULT
+      promoter: TO_BSON_CONVERSIONS[this.props.type.bson_type] || DEFAULT
     });
 
-    if (TO_JS_CONVERSIONS.hasOwnProperty(this.props.type.bsonType)) {
-      d3.select(el)
-        .datum(TO_JS_CONVERSIONS[this.props.type.bsonType](this.props.type.values))
-        .call(this.state.chart);
-    } else {
-      d3.select(el)
-        .datum(this.props.type.values)
-        .call(this.state.chart);
-    }
+    d3.select(el)
+      .datum(this.props.type.values)
+      .call(this.state.chart);
```

Comment:
I am having a few issues with how these charts are getting rendered. On master, for example, this graph is a horizontal bar graph, but with this version it's vertical. This happens with a few of the fields, and not just with `ships.shipwrecks`.

<img width="1422" alt="d3 creates a vertical bar chart on this branch when master has this as horizontal" src="https://user-images.githubusercontent.com/8107784/66204924-579a3c80-e6ac-11e9-837c-df91785e88b0.png">

The other issue is with the vertical bar chart in the `chart` field in this database. Where specifically do we set the padding between values? This field renders like this, but master has this very nice padding that makes the chart a lot more readable.
<img width="1433" alt="Chart field is missing padding" src="https://user-images.githubusercontent.com/8107784/66205054-a942c700-e6ac-11e9-9c74-7466227a79f3.png">

The same style of chart renders fine, for example, for `depth`'s `Double` values, so I am not super sure what I have to do to make it the same (or if it's actually a backend issue that I have to fix):
<img width="1431" alt="Depth's Double field renders fine" src="https://user-images.githubusercontent.com/8107784/66205152-f030bc80-e6ac-11e9-8622-b22e5ce1c018.png">